docs/getting-started.md

```diff
@@ -54,7 +54,7 @@ directory (`/usr/local/bin`):
 > You can find other download link in the release page that matches your machine.
 
 ```shell
-version="0.4.1"
+version="0.5.0"
```

Comment:
unrelated change for this PR? :)

Reply:
This is intentionally part of the release process. We should probably make the commit message clearer to reflect this, though, or else make it a separate commit.